ui/component/app/view.jsx

```diff
@@ -24,7 +24,6 @@ import LANGUAGES from 'constants/languages';
 import { BeforeUnload, Unload } from 'util/beforeUnload';
 import { platform } from 'util/platform';
```

Comment:
- [ ] Enhancement:  Improve chunking
    - Few pages like publishes can probably be chunked into 1 file to reduce traffic.